bindings/go/README.md

```diff
@@ -53,7 +53,7 @@ In order to build:
    (or install pre-built motr-devel package).
  2. Install Go: `sudo yum install go`.
  
-Note: Go version 1.16 or above is required to build this project.
+Note: Go version 1.11 or above is required to build this project.
```

Comment:
I think by this time everyone is using at least go-1.15, so this note is needless and can be dropped.